src/elements/header-toolbar.html

```diff
@@ -134,30 +168,27 @@
         ></paper-icon-button>
       </div>
       <div layout horizontal center flex>
-        <a class="toolbar-logo" href="/" hidden$="[[!viewport.isLaptopPlus]]" layout horizontal title="{$ title $}"></a>
+        <a class="toolbar-logo" href="/" layout horizontal title="{$ title $}"></a>
       </div>
 
-      <paper-tabs
+      <div
```

Comment:
I couldn't figure out how to get the existing menubar/tabs setup working 😞 so I just switched it to a more typical list of anchors in a div. Ah well.

This also changes the menu to always have the white background, so the top-left event logo is always visible.